libr/core/rtr.c

```diff
@@ -659,26 +659,19 @@ static int r_core_rtr_http_run(RCore *core, int launch, const char *path) {
 						char *out, *cmd = rs->path + 5;
 						r_str_uri_decode (cmd);
 						r_config_set (core->config, "scr.interactive", "false");
-						if (!strcmp (cmd, "=h*")) {
-							if (r_sandbox_enable (0)) {
-								out = NULL;
-							} else {
-								/* do stuff */
-								out = NULL;
-								r_socket_http_close (rs);
-								free (dir);
-								free (refstr);
-								ret = -2;
-								goto the_end;
-							}
-						}
-						if (*cmd == ':') {
+
+						if (!strcmp (cmd, "=h*") ||
+						    !strcmp (cmd, "=h--") &&
```

Comment:
add one more nesting of parenthesis here around the =-- and the sandbox check or the two strcmps